jax/_src/scipy/special.py

```diff
@@ -2201,6 +2230,115 @@ def ci_series(x):
 
   return si, ci
 
+
+def _sici_asympt(x: ArrayLike):
+  s = jnp.sin(x)
+  c = jnp.cos(x)
+  z = 1.0 / (x * x)
+
+  # Values come from Cephes Implementation used by Scipy https://github.com/jeremybarnes/cephes/blob/60f27df395b8322c2da22c83751a2366b82d50d1/misc/sici.c
+  FN4 = jnp.array([
```

Comment:
Please use `np.array` rather than `jnp.array` for lists of python floats; `jnp.array(list)` can be very slow.